package/src/hooks/useViewport.ts

```diff
@@ -25,11 +25,13 @@ export const useViewport = (rounded?: boolean) => {
     return () => subscriptions?.remove();
   }, []);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
are these false positives? curious why the script detects these as hook deps :)

Reply:
That is, assuming some downstream project doesn't actually rely on the fact that these aren't memoized :D that might be a bit tricky

Reply:
I think it would be best if we remove `useMemo(() => ({ vh, vw }), [vh, vw]);` and return directly.

Reply:
Yep, after I'm done with some other things I'll jump into taking care of these easier low hanging fruits which should improve performance a bit.

Reply:
nice catch here, this useMemo does nothing here, useCallback for both vw and vh and then remove useMemo here

I think vw and vh is used in multiple React.Memos

Reply:
Yep, I'll have to look how far the non-memoization cascades here. Also I hope that some downstream implementations do not do not rely on the fact that `vh` and `vw` aren't memoized and that we break them. Should we add this somewhere ?